packages/content-conformance/src/config.ts

```diff
@@ -64,6 +65,10 @@ export async function loadConfig({
 
   config.root = resolveConfigRoot(config.root, cwd)
 
+  if (!config.partialsDirectory) {
+    config.partialsDirectory = 'content/partials'
```

Comment:
Is this a baked in default?

Reply:
Yeah, it seemed like a reasonable default. Think we should not do it here?

Reply:
I think here is OK, but also feel like a central file/export of **all-things-defaults** could be useful for preventing _default config sprawl_ (very loose opinion)

Reply:
Additionally, curious if this needs to account for 'legacy' `pages/partials` 
- https://github.com/hashicorp/vault/tree/v1.4.0/website/pages/partials

Reply:
Good callout re legacy partials. I'm thinking probably not, but if it turns out that folks are editing content that far back we might want to handle it.

Reply:
@thiskevinwang I abstracted out the default values into a constant at the top of `config.ts`, but holding off on a separate file until we start introducing more default values used throughout the package.